connect/chains/config.go

```diff
@@ -0,0 +1,87 @@
+package chains
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	igniteconfig "github.com/ignite/cli/v28/ignite/config"
+
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	configName = "connect.yaml"
+
+	// ErrConfigNotFound is returned when the config file is not found
+	ErrConfigNotFound = fmt.Errorf("config file not found")
+)
+
+type Config struct {
+	Chains map[string]*ChainConfig `yaml:"chains"`
+}
+
+type ChainConfig struct {
+	ChainID      string `yaml:"chain_id"`
+	Bech32Prefix string `yaml:"bech32_prefix"`
+	GRPCEndpoint string `yaml:"grpc_endpoint"`
+}
+
+func (c *Config) Save() error {
+	out, err := yaml.Marshal(c)
+	if err != nil {
+		return fmt.Errorf("failed to marshal config: %w", err)
+	}
+
+	configDir, err := ConfigDir()
+	if err != nil {
+		return err
+	}
+
+	connectConfigPath := path.Join(configDir, configName)
+	if err := os.WriteFile(connectConfigPath, out, 0644); err != nil {
```

Comment:
Octal integer literals should use the 0o prefix on modules using Go 1.13 and later
```suggestion
	if err := os.WriteFile(connectConfigPath, out, 0o644); err != nil {
```